README.md

```diff
@@ -35,23 +35,24 @@ You can also see this component in action at [voicerecordpro.com](https://www.vo
 ## Example
 
 ```js
-import ReactMic from 'react-mic';
+import { ReactMic } from 'react-mic';
 
-class Example extends React.Component {
+export class Example extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       record: false
-    }
+    };
+    this.onClick = this.startRecording.bind(this);
```

Comment:
Hi,

startRecording and stopRecording are defined using  es6 "fat" arrow functions.  Therefore you don't need to bind them to the Example class in the constructor.  The "this" keyword will have the correct scope when startRecording & stopRecording are defined this way:

```
startRecording= () => { }
stopRecording= () => { }
```

Can you remove these lines in your PR?